Makefile

```diff
@@ -1,8 +1,18 @@
+.PHONY: test-in-container
+
+TEST_IMAGE=cwt-tests
 TEST_DIR=test/$(TARGET)
-TESTS=$(shell ls $(TEST_DIR)/test_* | xargs basename -s .py | xargs)
+TESTS=$(shell cd test/ && ls test_*)
+TEST_CONTAINER_RUN=docker run --rm -it -v $(CURDIR):/cwt $(TEST_IMAGE)
```

Comment:
Can we add the current dir content into the image directly (in Dockerfile) instead of mounting it as a volume?